packages/nouns-subgraph/src/nouns-dao.ts

```diff
@@ -100,6 +109,35 @@ export function handleProposalCreatedWithRequirements(
   proposal.save();
 }
 
+export function handleProposalUpdated(event: ProposalUpdated): void {
+  const updateId = event.params.id
+    .toString()
+    .concat('-')
+    .concat(event.transaction.hash.toHexString());
```

Comment:
@eladmallel unlikely to have collisions, but safer to also concat `event.logIndex.toString()`

Reply:
added